src/assert.js

```diff
@@ -14,6 +14,14 @@ class Assert {
 
 	// Assert helpers
 
+	timeout( duration ) {
+		if ( objectType( duration ) !== "number" ) {
```

Comment:
For 99% of cases it's probably fine, but if someone were to pass a `new Number(1000)`, we'd miss it, even though it is a `number`.